src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -2869,7 +2973,7 @@ private void ClassifyUpdate(AttributeListSyntax oldNode, AttributeListSyntax new
             {
                 if (!SyntaxFactory.AreEquivalent(oldNode.Target, newNode.Target))
                 {
-                    ReportError(RudeEditKind.Update);
+                    ReportError(RudeEditKind.Update, spanNode: ((SyntaxNode)newNode.Target) ?? newNode);
```

Comment:
This is the maintain existing functionality with the change in `GetDiagnosticSpan`